recipes/opentdf-client/all/conanfile.py

```diff
@@ -38,7 +38,7 @@ def _minimum_cpp_standard(self):
     @property
     def _minimum_compilers_version(self):
         return {
-            "Visual Studio": "17" if Version(self.version) < "1.1.5" else "16",
+            "Visual Studio": "17" if Version(self.version) < "1.1.5" else "15",
             "msvc": "19.22",
```

Comment:
This value is wrong by the way and there is no condition like for Visual Studio, why?